src/util/Util.js

```diff
@@ -248,9 +248,10 @@ Util.executeOnVisiblePage = function(cb) {
   }
 };
 
-Util.isAndroidOVEnrollment = function() {
-  const ovEnrollment = decodeURIComponent(window.location.href).includes(ovDeepLink);
-  return BrowserFeatures.isAndroid() && ovEnrollment;
+Util.isAndroidOVEnrollment = function(authentication) {
+  const redirectUri = authentication?.request?.redirect_uri;
+  const ovEnrollment = redirectUri?.includes(ovDeepLink);
+  return BrowserFeatures.isAndroid() && !!ovEnrollment;
```

Comment:
Question: 
Alternatively we can consider checking `app.value.name === 'Okta_Authenticator'` in [IDX response](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/terminal-okta-verify-enrollment-android-device.json#L45) instead of checking `authentication.value.request.redirect_uri.includes('https://login.okta.com/oauth/callback')`

Should we?